src/Arcus.Templates.AzureFunctions.Http/OrderFunction.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.Threading.Tasks;
+using Arcus.Security.Core;
+using Arcus.WebApi.Logging.Correlation;
+using GuardNet;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Arcus.Templates.AzureFunctions.Http.Model;
+
+namespace Arcus.Templates.AzureFunctions.Http
+{
+    /// <summary>
+    /// Represents the root endpoint of the Azure Function.
+    /// </summary>
+    public class OrderFunction : AzureFunction
+    {
+        private readonly ISecretProvider _secretProvider;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="OrderFunction"/> class.
+        /// </summary>
+        /// <param name="secretProvider">The instance that provides secrets to the HTTP trigger.</param>
+        /// <param name="httpCorrelation">The instance to handle the HTTP request correlation.</param>
+        /// <param name="logger">The logger instance to write diagnostic trace messages while handling the HTTP request.</param>
+        /// <exception cref="ArgumentNullException">Thrown when the <paramref name="secretProvider"/> or <paramref name="httpCorrelation"/> is <c>null</c>.</exception>
+        public OrderFunction(ISecretProvider secretProvider, HttpCorrelation httpCorrelation, ILogger<OrderFunction> logger) : base(httpCorrelation, logger)
+        {
+            Guard.NotNull(secretProvider, nameof(secretProvider), "Requires a secret provider instance");
+            _secretProvider = secretProvider;
+        }
+
+        [FunctionName("order")]
+        [RequestSizeLimit(100)]
```

Comment:
Let's leave this out of the template for now

Reply:
It was to show the possibilities, but maybe too much clutter indeed.

Reply:
I'll remove this request size limit attribute. Resolving.